config/settings/travis.py

```diff
@@ -15,7 +15,7 @@
         'USER': 'postgres',
         'PASSWORD': '',
         'HOST': 'localhost',
-        'PORT': '5432',
+        'PORT': '5433',
```

Comment:
I'm guessing that this is here because I cherry picked the change into bricr-dev from develop